app/src/main/java/com/dscnsut/app/blog_content.kt

```diff
@@ -0,0 +1,59 @@
+package com.dscnsut.app
```

Comment:
Make sure you follow the file naming schemes. Any kotlin file name should be in camel case and the first letter should be capital. Also, any fragment file name should end with keyword "Fragment", which in this case would be BlogContentFragment.